sdk/storage/azure-storage-common/inc/azure/storage/common/crypt.hpp

```diff
@@ -3,17 +3,21 @@
 
 #pragma once
 
+#include <azure/core/base64.hpp>
+
```

Comment:
storage code has an unwritten rule of the order of these include headers. Usually we put this header after C++ standard library headers

Reply:
basically @katmsft and I want to follow https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes because our guidelines lack such a rule.

I totally understand if you don't like the rule from Google C++ style guide. But before we add a new regulation into our guidelines, can we just make it consistent within storage sdk?

Reply:
Yes, makes sense. Already updated to fix that :)